rst_files/linear_algebra.rst

```diff
@@ -92,22 +96,16 @@ Activate the project environment, ensuring that ``Project.toml`` and ``Manifest.
     using Plots
 
     vecs = ([2, 4], [-3, 3], [-4, -3.5])
-    x_vals = zeros(2, length(vecs))
-    y_vals = zeros(2, length(vecs))
-    labels = []
 
     # Create matrices of x and y values, labels for plotting
-    for i ∈ eachindex(vecs)
-        v = vecs[i]
-        x_vals[2, i] = v[1]
-        y_vals[2, i] = v[2]
-        labels = [labels; (1.1 * v[1], 1.1 * v[2], "$v")]
-    end
+    x_vals = reduce(hcat, ([0, v] for v in first.(vecs)))
```

Comment:
See other comments.  Is there any way to make the `x_vals` and `y_vals` constructions easier to understand?  If not, I would rather stick with the for loop in this case.